### ISSUE TITLE
Enemies stuck in top of Barsuk bow

### ISSUE THREAD

**Issue by Koku0** [Regalis11/Barotrauma#10698]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Enemies get stuck on the top of the bow of the Barsuk, which means that they are unable to be hit by any ship-based weaponry and continually attack the submarine. The enemies are also unable to retreat and attack once more, as they normally do.

### Reproduction steps

1. Start a campaign, multiplayer or singleplayer
2. Use the Barsuk submarine
3. Get attacked and watch as enemies are permanently stuck in the top of the bow of the submarine.

### Bug prevalence

Happens regularly

### Version

Faction/endgame test branch

### Which operating system did you encounter this bug on?

Windows

**Comment by Koku0** [comment]
Mostly crawlers. This might be me misunderstanding their behaviour, as I've not seen this before (playing since Dec 2021). Was this behaviour added recently?

**Comment by Regalis11** [comment]
No, this behavior should've been in the game even before it was released on Steam. I guess it could be there's been some issues in the AI that have prevented it from happening somewhere down the line.
